src/pages/groupComparison/pathwayMapper/GroupComparisonPathwayMapper.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from 'react';
+import PathwayMapperTable, {
+    IPathwayMapperTable,
+} from '../../../shared/lib/pathwayMapper/PathwayMapperTable';
+import { observer } from 'mobx-react';
+import autobind from 'autobind-decorator';
+import { computed, makeObservable, observable } from 'mobx';
+import { Row } from 'react-bootstrap';
+
+import PathwayMapper from 'pathway-mapper';
+
+import 'pathway-mapper/dist/base.css';
+import 'cytoscape-panzoom/cytoscape.js-panzoom.css';
+import 'cytoscape-navigator/cytoscape.js-navigator.css';
+import { AlterationEnrichmentRow } from 'shared/model/AlterationEnrichmentRow';
+import { ComparisonGroup } from '../GroupComparisonUtils';
+import { GenesSelection } from 'pages/resultsView/enrichments/GeneBarPlot';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import GroupComparisonStore from '../GroupComparisonStore';
+import { GeneOptionLabel } from 'pages/resultsView/enrichments/EnrichmentsUtil';
+
+interface IGroupComparisonPathwayMapperProps {
+    genesOfInterest: AlterationEnrichmentRow[];
+    activeGroups: ComparisonGroup[] | undefined;
+    genomicData: AlterationEnrichmentRow[];
+    store: GroupComparisonStore;
+}
+
+@observer
+export default class GroupComparisonPathwayMapper extends React.Component<
+    IGroupComparisonPathwayMapperProps
+> {
+    clearMessage: (() => void) | undefined;
+
+    constructor(props: IGroupComparisonPathwayMapperProps) {
+        super(props);
+        makeObservable(this);
+
+        // @ts-ignore
+        import(/* webpackChunkName: "pathway-mapper" */ 'pathway-mapper').then(
+            (module: any) => {
+                this.PathwayMapperComponent = (module as any)
+                    .default as PathwayMapper;
+            }
+        );
+    }
+
+    @observable.ref PathwayMapperComponent:
+        | PathwayMapper
+        | undefined = undefined;
+    @observable isGeneSelectionPopupVisible: boolean = false;
+    @observable _genesOfInterest: string[] | undefined = undefined;
+
+    @computed get selectedGenes() {
+        return (
+            this._genesOfInterest ||
+            this.props.store.genesWithMaxFrequency.map(g => g.hugoGeneSymbol)
+        );
+    }
+
+    @computed get genesOfInterest(): AlterationEnrichmentRow[] {
```

Comment:
seems strange that we call an array of AlterationEnrichmentRow "genesOfInterest" 
also seems to conflict with the selected/active gene members above

Reply:
renamed to `alterationEnrichmentRowData` for clarity